### PR TITLE
use int ceil/floor div

### PR DESCRIPTION
**Pull request by Yutyrannus**

### CODE REVIEW

File: src/layer/pooling.cpp [hunk 1]
```diff
@@ -119,12 +118,16 @@ int Pooling::forward(const Mat& bottom_blob, Mat& top_blob, const Option& opt) c
 
                 for (int i = 0; i < out_h; i++)
                 {
-                    const int ih0 = floor(((float)h / out_h) * i);
-                    const int ih1 = ceil(((float)h / out_h) * (i + 1));
+                    // floor div
+                    const int ih0 = h / out_h * i;
+                    // ceil div
+                    const int ih1 = (h + out_h - 1) / out_h * (i + 1);
```
Comment (nihui) [inline src/layer/pooling.cpp:124]:
```
                    // floor div
                    const int ih0 = h * i / out_h;
                    // ceil div
                    const int ih1 = (h * (i + 1) + out_h - 1) / out_h;
```

**Review by nihui — CHANGES_REQUESTED** [review]
and please fix the remaining

File: src/layer/pooling.cpp (hunk 1 above)
Comment (Yutyrannus) [reply src/layer/pooling.cpp]: fix

**Review by nihui — APPROVED** [review]

**Comment by nihui** [comment]
Thanks for your contribution !

**Pull request merged**